### ISSUE TITLE
Feature Request: Line/XY scatter plot, multiple X & Y axis & Stacked chart 

### ISSUE THREAD

**Issue by priyankguddu** [man-group/dtale#442]
Thanks for the D-tale, now my daily goto tool for last 1 week of exploration. While I enjoy charting a lot, it would be great to have a standard dashboard tool too. With following minor modification, it can just enhance overall capability as multiuser, multi charting tool for EDA. 
**_Some Enhancement request:_**

- We can put multiple variable in Y axis and it plots single chart showing both, can we have feature split into two charts ? (similar to grouping where we can see different charts)
-  In XY plot (It would be good to have feature to connect the dots, or plot like a continuous line)
- Add Multiple XY chart with user option from dropdown to select X axis colm  and Y-axis from same data frame 
- Although for memory constraint only limited DF can be loaded, which is completely understandable, but if a new Host (say Host2) connects to client (localhost:40000), can we by default give page to load new data frame? (i.e open with /2 in mapping and so ON) so that a user can load and work on the new data frame in parallel without having to see the default /1 (Original Host -1)   dynamic view, If host-1 wants to share the Host-2 the same DF, they can share the URL
- Some column if the there are floating point numbers, the click to column(describe)/analysis menu doesn't popup

Again great tool for EDA and code export feature is cool. Will wait if these feature u can approve

**Comment by aschonfeld** [comment]
Thanks for the feedback, this is great!
* This should be pretty simple by adding another toggle in addition to "By Group" for "By Y Value" (or something better sounding 😄 )
* So in scatter plots you can view the trendline, but otherwise this just sounds like a line chart?  If you could show me an example it will help
* This is a tougher request, but I can certainly look into how tough it would be to update the UI for building multiple charts
* Sure, that's easy enough to show a list of what dataframes are available rather than forcing them to the piece of data with the lowest data_id (in most instances 1)
  * as for sharing the URLs, that should already exist with the "Popup Chart" or "Copy Link" buttons that is at the top of each chart.  If you click "Copy Link" then it creates a link which if pasted into a browser will show others the chart you have generated.
* Can you give me a little more detail?  If you go to the "Describe" tab can you tell me what data type is associated with the float columns which don't get a "Column Analysis" chart
Thanks for your help

**Comment by priyankguddu** [comment]
Thanks for the response, I normally go to VEUSZ https://veusz.github.io/ (amazing but desktop only tool) on Numpy and Qt 
For 2&3) Scatter Line with connected dots, here we have 2 sensor data coming at 2 different time stamp but plotted in single chart 
i.e each chart has independent x and y which we can pull from the data frame, mostly its monotonic values that goes in X axes where Y is plotted against, also see here the scattor plot can be put into connected lines 
![image](https://user-images.githubusercontent.com/25154108/109091499-c3c67780-76c9-11eb-8952-279d9e13b75d.png)

For 5th issue, I have some 10000 rows and 200 colm, some colm it Describe and colmn tool comes some it doesn't popup, let me see if I can reporduce it, all colm dtype if float64, no string or int

**Comment by aschonfeld** [comment]
@priyankguddu I get what you're saying now for 2) & 3)

So I think this will be achievable once I apply the "chart per group" logic to y-axes as well and then you can just select the line chart for your data.

**Comment by aschonfeld** [comment]
Just wanted to give you an update on this. I've made a lot of progress.  Here is a demo with the following functionality:
* chart per y
* switching between pieces of data in D-Tale
* saving a chart and then building another one from another piece of data in D-Tale

https://user-images.githubusercontent.com/11547371/110421049-acdd3900-806a-11eb-96a9-0d56514a7bbc.mov

**Comment by aschonfeld** [comment]
@priyankguddu I have just released a new version 1.38.0 with the updates in that demo.  If you can get me some more information on your 5th point please open another issue or for anything I may have missed for that matter. 🙏
